pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -376,11 +376,16 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 	}
 
 	var sgRules = []sgRule{
-		{podSgId, "0.0.0.0/0", -1, -1, rules.ProtocolTCP},
+		{podSgId, "0.0.0.0/0", 0, 0, rules.ProtocolTCP},
 		{masterSgId, openStackSvcCIDR, etcdPort, etcdPort, rules.ProtocolTCP},
 		{masterSgId, openStackSvcCIDR, apiPort, apiPort, rules.ProtocolTCP},
 	}
 
+	var decommissionedRules = []sgRule{
+		{podSgId, workerSubnet.CIDR, 0, 0, ""},
```

Comment:
shouldn't be the other way around? removing the access from the pods to the workers?

Reply:
This is supposed to remove [this rule](https://github.com/openshift/cluster-network-operator/pull/465/files#diff-4ab6aa530ac077ca991bba003d6ea267L1016) that got replaced by [this rule](https://github.com/openshift/cluster-network-operator/pull/465/files#diff-4ab6aa530ac077ca991bba003d6ea267R1017).

Reply:
ok